### PR TITLE
Force returned value to be float64

### PR DESCRIPTION
**Pull request by alvarocabanas**
### What are you changing/fixing?

This change adds a method to require the value returned to be casted to float64

### Does your Pull Request introduce breaking changes?

No

### Do the users need to upgrade immediately to the new version?

No

### Do you introduce new dependencies on other libraries?

No

## Checklist: before you submit

- [ x ] apply the labels that best suit the context of your Pull Request.
- [ x ] include unit or integration testing for your changes/additions.

### CODE REVIEW

File: gojmx/types.go [hunk 1]
```diff
@@ -57,6 +62,38 @@ func (j *AttributeResponse) GetValue() interface{} {
...
+		}
+		return parsedValue, nil
+	case ResponseTypeDouble:
+		return j.DoubleValue, nil
+	case ResponseTypeInt:
+		return float64(j.IntValue), nil
+	case ResponseTypeErr:
+		return 0, nil
+	default:
+		panic(fmt.Sprintf("unkown value type: %v", j.ResponseType))
```
Comment (roobre) [inline gojmx/types.go:89]: Can we return an error here, instead of panicking?

Comment (alvarocabanas) [reply gojmx/types.go:89]: This is how it's done in the other GetValue, I am replying the same behaviour.

Comment (cristianciutea) [reply gojmx/types.go:89]: Is a valid case of panicking, the types are defined in the protocol and should only panic when the java module and go module doesn't respect the procol

File: gojmx/types.go [hunk 2]
```diff
@@ -57,6 +62,38 @@ func (j *AttributeResponse) GetValue() interface{} {
...
+		if j.BoolValue {
+			return 1, nil
+		}
+		return 0, nil
+	case ResponseTypeString:
+		parsedValue, err := strconv.ParseFloat(fmt.Sprintf("%v", j.StringValue), 64)
+		if err != nil {
+			return 0, err
+		}
+		if isNaNOrInf(parsedValue) {
```
Comment (cristianciutea) [inline gojmx/types.go:78]: i think Nan values/ inf etc are handled by the SDK. Library can return them

Comment (alvarocabanas) [reply gojmx/types.go]: Ok, i'll remove this.

Comment (alvarocabanas) [reply gojmx/types.go]: Fixed in https://github.com/newrelic/nrjmx/pull/130/commits/eea0462dc1a5b686750c4c07de429a62606a5996

File: gojmx/types.go (hunk 1 above)
Comment (alvarocabanas) [reply gojmx/types.go:82]: noted

**Review by cristianciutea — APPROVED** [review]
LGTM, thank you!

**Pull request merged**